scripts/integration_test_contracts.sh

```diff
@@ -1,72 +1,78 @@
 #!/bin/bash
 
+set -e
+
 ## parameters
-CONTRACT_ADDRESS='link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8'
 FROM_ACCOUNT='link146asaycmtydq45kxc8evntqfgepagygelel00h'
 URL='http://localhost:26657'
 CHAIN_OPTION='--chain-id finschia --keyring-backend test -b block -o json -y'
+COLLECTION_CONTRACT=${COLLECTION_CONTRACT:-collection.wasm}
 
 ## Wait until the height of the block is greater than or equal to 1.
-timeout 60 bash -c 'until [[ $(curl -s "http://localhost:26657/status?" | jq -r ".result.sync_info.latest_block_height // 0") -ge 1 ]]; do sleep 0.5; done'
+echo "##### Waiting finschia node starts #####" >&2
+timeout 60 bash -c 'until [[ $(curl -s "'${URL}'/status?" | jq -r ".result.sync_info.latest_block_height // 0") -ge 1 ]]; do sleep 0.5; done'
+echo "OK" >&2
 
 # If the timeout fails, the process will be terminated abnormally.
 exitstatus=$?
 if [[ $exitstatus -ne 0 ]]; then
-    echo "waiting finschia node is timeout" >&2
-    exit 1
+  echo "waiting finschia node is timeout" >&2
+  exit 1
 fi
-# send transaction to node
-cur_path=`pwd`
-
-artifacts_path="artifacts"
 
 # This is a function that checks if executeMsg is successful.
 check_run_info() {
   local res="$1"
   local msg="$2"
-	if [[ "$res" == *"failed"* ]]; then
-		echo -e "$msg$res" >&2
-		exit 1
-	fi
+  if [[ "$res" == *"failed"* ]]; then
+    echo -e "$msg$res" >&2
+    exit 1
+  else
+    echo "===PASS===" >&2
+  fi
 }
 
-#*** store collection contract ***
-result=$(fnsad tx wasm store ${artifacts_path}/collection.wasm --node ${URL} --from ${FROM_ACCOUNT} --gas 10000000 ${CHAIN_OPTION})
+# send transaction to node
+
+echo "##### TEST: store collection contract #####" >&2
+result=$(fnsad tx wasm store ${COLLECTION_CONTRACT} --node ${URL} --from ${FROM_ACCOUNT} --gas 10000000 ${CHAIN_OPTION})
 
 ## confirm a result of `store`
 raw_log=$(echo ${result} | jq .raw_log)
 check_run_info "${raw_log}" "store: "
 
-#*** instantiate collection contract ***
+echo "##### TEST: instantiate collection contract #####" >&2
 CODE_ID=$(echo "${result}" | jq '.logs[] | select(.msg_index == 0) | .events[] | select(.type == "store_code") | .attributes[] | select(.key == "code_id") | .value | tonumber')
-result=$(fnsad tx wasm instantiate ${CODE_ID} '{"name":"collection_name","uri":"collection_uri","meta":"collection_meta", "owner":"'${CONTRACT_ADDRESS}'"}' --label collection1 --admin ${FROM_ACCOUNT}  --from ${FROM_ACCOUNT} --node ${URL} ${CHAIN_OPTION})
+result=$(fnsad tx wasm instantiate ${CODE_ID} '{"name":"collection_name","uri":"collection_uri","meta":"collection_meta"}' --label collection --admin ${FROM_ACCOUNT}  --from ${FROM_ACCOUNT} --node ${URL} ${CHAIN_OPTION})
```

Comment:
It seems no `contract_address` in the env_variables because instantiate hasn't been finished.
And I want to ask why the value of `owner` is `contract_address` not `admin_address`

Reply:
> It seems no contract_address in the env_variables because instantiate hasn't been finished.
exactly. So this PR removes it from the contract's init msg and removes the contract address const from this script.

> And I want to ask why the value of owner is contract_address not admin_address
It was a bug and this PR fixes it.

Reply:
I see, but isn't it should be `FROM_ACCOUNT`
```
result=$(fnsad tx wasm instantiate ${CODE_ID} '{"name":"collection_name","uri":"collection_uri","meta":"collection_meta", "owner":"'${FROM_ACCOUNT}'"}' --label collection --admin ${FROM_ACCOUNT} --from ${FROM_ACCOUNT} --node ${URL} ${CHAIN_OPTION})
```

Reply:
No, it is not in this test. If we set the owner as `FROM_ACCOUNT`, this contract cannot do something to the collection without approval from the `FROM_ACCOUNT`. This test confirms that a contract (in this case, collection.wasm) can be an owner of the collection and this contract operates to the collection as an owner.